modules/RouterUtils.js

```diff
@@ -1,26 +1,7 @@
-import deprecateObjectProperties from './deprecateObjectProperties'
-
 export function createRouterObject(history, transitionManager) {
   return {
     ...history,
     setRouteLeaveHook: transitionManager.listenBeforeLeavingRoute,
```

Comment:
We can add it later if it's needed. I thought once upon a time that it'd make it easier to integrate with Redux or something, but now I'm thinking that https://github.com/reactjs/react-router/issues/3190 is a better approach – for some day.